File: dogwalkingui/src/components/BookingDetails/Components/AboutYourDog/Dog/Components/Size.js
import SelectRow from './Components/SelectRow';

function Size(props) {

  return (
    <SelectRow
      label='Size'
      baseProps={props.baseProps}
      propertyName='size'
      options={<>
        <option>Please select</option>
        <option>Toy</option>
        <option>Small</option>
        <option>Medium</option>
        <option>Large</option>
      </>
      }
    />
  )
}

export default Size;

File: api/Src/Enums/Size.cs
using System.Runtime.Serialization;

namespace DogWalkingApi.Enums
{
    public enum Size
    {
        [EnumMember(Value = "Toy")]
        Toy,
        [EnumMember(Value = "Small")]
        Small,
        [EnumMember(Value = "Medium")]
        Medium,
        [EnumMember(Value = "Large")]
        Large
    }
}
